src/Sleep.js

```diff
@@ -0,0 +1,50 @@
+class Sleep {
+  constructor(userId, sleepData) {
+    this.sleepDataPerUser = sleepData.filter(data => data.userID === userId);
+    this.sleepData = sleepData;
+  }
+
+  // getAverage(minDate, maxDate, numDays) {
+  //   const average = array => array.reduce((a, b) => a + b) / array.length;
+  //   console.log(average([1,2,3,4,5]));
+  // }
+
+  getSleepDataByGivenDay(date, type) {
+    const dailySleep = this.sleepDataPerUser.reduce((acc, data) => {
+      if (data.date === date) {
+        acc = data[type];
+      }
+      
+      return acc;
+    }, 0);
+
+    return dailySleep;
+  }
+
+  getAvgSleepData(type, dataSet) {
+    let sum, average;
+    sum = dataSet.reduce((acc, data) => {
+      if (data[type]) {
+        acc += data[type];
+      }
+
+      return acc;
+    }, 0);
+
+    average = sum / dataSet.length;
+    return Number(average.toFixed(1));
```

Comment:
Can you change the return to Math.round(sum / dataSet.length) to dry this function up a little?